### ISSUE TITLE
initialDirectory not working

### ISSUE THREAD

**Issue by julianklose** [miguelpruivo/flutter_file_picker#970]
**Describe the bug**
I'm using the newest version (4.4.0) of the file_picker including the new `initialDirectory` option. I tried using it with the directory picker. Unfortunately, it doesn't seem to work, I always start at my home directory.

**Platform**

- [ ] Android
- [ ] iOS
- [ ] Web
- [x] Desktop

**Platform OS version**
Windows 10

**How are you picking?**
```
var dir = await FilePicker.platform.getDirectoryPath(
        lockParentWindow: true, initialDirectory: Directory.current.path);
```

**Flutter Version details**
[√] Flutter (Channel stable, 2.10.1, on Microsoft Windows [Version 10.0.19044.1526], locale de-DE)
    • Flutter version 2.10.1 at C:\Users\JulianKlose\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision db747aa133 (11 days ago), 2022-02-09 13:57:35 -0600
    • Engine revision ab46186b24
    • Dart version 2.16.1
    • DevTools version 2.9.2

[√] Android toolchain - develop for Android devices (Android SDK version 32.1.0-rc1)
    • Android SDK at C:\Users\JulianKlose\AppData\Local\Android\sdk
    • Platform android-32, build-tools 32.1.0-rc1
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Build Tools 2019 16.11.9)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools
    • Visual Studio Build Tools 2019 version 16.11.32106.194
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 2021.1)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)

[√] VS Code (version 1.64.2)
    • VS Code at C:\Users\JulianKlose\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.34.0

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19044.1526]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 98.0.4758.102
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 96.0.1054.62

[√] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!

**Comment by julianklose** [comment]
I just read in the merge request, that it doesn't work with the windows directory picker. Maybe it will with #915. Can you add a notice about this in the documentation until it works?

**Comment by philenius** [comment]
Hi @julianklose , thank you for noticing this. The Dart docs will be fixed with #915 and I already updated [the API section of our wiki](https://github.com/miguelpruivo/flutter_file_picker/wiki/API).

Fortunately, Microsoft's `IFileDialog` API supports setting the initial directory even when picking a directory. However, I don't understand how to use it. So, setting the initial directory for the directory picker dialog is still not possible in `file_picker`. Sorry for the inconveniences.
I added a TODO to our code for the missing implementation and I'll close this issue until I or someone else finds out how to use the `SetFolder()` / `SetDefaultFolder()` methods.
